lib/dry/logic/predicates.rb

```diff
@@ -148,11 +148,19 @@ def self.included(other)
       end
 
       predicate(:includes?) do |value, input|
-        input.include?(value)
+        begin
+          case input
+          when String, Range, Array, Hash then input.include?(value)
```

Comment:
That sounds nicer. Also makes it more flexible with custom classes.
I also really hate rescuing the type error but if couldn't think of a better way...